### PR TITLE
Feature/clear all added button

### LINKED ISSUE #117: Clearing added items in query doesn't make them reappear in search results
#### Description
Clearing added items in query doesn't make them reappear in search results

#### Screenshots
issue-117

#### To Reproduce
* add all diseases to query
* click the "clear" button in the Disease Type Set column
* notice how results remain empty

### PR DESCRIPTION
**Pull request by ramenhog**
## Issue Number
Closes #117 

## Purpose/Implementation Notes
* added the ability to clear all selected added params in the queryParamSelector

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Functional tests
* Selecting params from the table in the "Added to Query" tab and clicking the "Remove from Query" removes the appropriate params from the query and repopulates search results
* Clicking the "X" on individual params in the "Overview" sidebar removes the appropriate param from the query and repopulates search results

## Checklist

_Put an `x` in the boxes that apply._

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Screenshots
pr-137

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -20,5 +20,16 @@
     <ui-view></ui-view>    
    
     <script src="js/main.js"></script>
+    <!-- Google Analytics -->
+    <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```
Comment (dongbohu) [inline app/index.html]: Can you make it more readable? This is hard to understand.

File: app/js/components/queryBuilder/queryParamSelector/queryParamSelector.tpl.html [hunk 2]
```diff
@@ -24,7 +24,7 @@ <h2>{{$ctrl.currentState()}} Selector</h2>
 		class="tabs__button"
 		ng-click="$ctrl.activeTab = 'search'"
 		ng-class="{'tabs__button--active': $ctrl.activeTab === 'search'}"
-	>Search Results ({{$ctrl.searchResults.length}})</button>
+	>Search Results</button>
```
Comment (dongbohu) [inline app/js/components/queryBuilder/queryParamSelector/queryParamSelector.tpl.html]: No need to show the number of results now?

**Comment by ramenhog** [comment]
@dongbohu sorry for all the GA code in here. I should have waited to merge that in here until after this was approved

File: app/js/components/queryBuilder/queryParamSelector/queryParamSelector.tpl.html (hunk 2 above)
Comment (ramenhog) [reply app/js/components/queryBuilder/queryParamSelector/queryParamSelector.tpl.html]: Yeah, users are more interested in the number of params being added to the query.

**Pull request merged**